makepkg.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import glob
+import os
+import os.path
+import shutil
+import subprocess
+import sys
+
+from azurelinuxagent.common.version import AGENT_NAME, AGENT_VERSION, AGENT_LONG_VERSION
+from azurelinuxagent.ga.update import AGENT_MANIFEST_FILE
+
+
+MANIFEST = '''{{
+    "name": "{0}",
+    "version": 1.0,
+    "handlerManifest": {{
+        "installCommand": "",
+        "uninstallCommand": "",
+        "updateCommand": "",
+        "enableCommand": "{1}",
+        "disableCommand": "",
+        "rebootAfterInstall": False,
+        "reportHeartbeat": False
+    }}
+}}'''
+
+ENABLE_CMD = "python -c 'from azurelinuxagent import agent; agent.main([\"-run-exthandlers\"]);' {0}"
```

Comment:
@ahmetalpbalkan It complains that "__main__.py" could not be found even though it is present. Any ideas?